cobra/core/model.py

```diff
@@ -414,37 +415,119 @@ def add_reactions(self, reaction_list):
         # from cameo ...
         self._populate_solver(reaction_list)
 
+    def add_to_problem(self, what):
+        """Add variables and constraints to the model's mathematical problem.
+
+        Useful for variables and constraints that can not be expressed with
+        reactions and simple lower and upper bounds. Additions are  reversed
+        upon exit if the model itself is used as context.
+
+        Parameters
+        ----------
+        what : list or tuple of optlang variables or constraints.
+           The variables or constraints to add to the model. Must be of
+           class `optlang.interface.Variable` or
+           `optlang.interface.Constraint`.
+        """
+        add_to_solver(self, what)
```

Comment:
In #404 I had to add a kwarg to `add_to_solver` when some function needed `sloppy=True`.. Maybe we'll need the same thing here?